### ISSUE TITLE
README conflicts with actual implementation

### ISSUE THREAD

**Issue by mlilback** [auth0/auth0-java-mvc-common#73]
The README states:

> Keep in mind that this library will not store any value for you, but you can use the SessionUtils class as a helper to store key-value data in the request's Session Storage.

I wish this were true. But AuthorizeUrl.java, lines 181-184:
```
        // Also store in Session just in case developer uses deprecated
        // AuthenticationController.handle(HttpServletRequest) API
        RandomStorage.setSessionState(request, state);
        RandomStorage.setSessionNonce(request, nonce);
```

This was giving me NPEs on my tests because I wasn't mocking a session. I'd prefer if you didn't force creation of a session, but that or the documentation needs to be changed.

**Comment by jimmyjames** [comment]
Thanks for raising this. That sentence from the `README` is misleading; we do not store any credentials from the returned `Tokens` object, but as you noted we do store the state and nonce in the session to handle the deprecation case noted in the comments. We'll update the `README` to be clearer.

**Comment by jimmyjames** [comment]
Closed with #78
